crates/biome_configuration/src/lib.rs

```diff
@@ -151,6 +153,11 @@ pub struct Configuration {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub html: Option<HtmlConfiguration>,
 
+    /// Specific configuration for the Markdown language
+    #[bpaf(external(markdown_configuration), optional)]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub markdown: Option<MarkdownConfiguration>,
```

Comment:
Please revert all the changes that involve the biome configuration. Doing this will expose the markdown parser **to our users** even though our parser isn't even ready yet. Without a working parser, we can't even provide functionalities such as formatter and/or linter.

If you wish to expose the markdown functionality as an experimental feature, then you would need to use a Rust feature. However, **it's too soon** for that.